lib/tokenize.cpp

```diff
@@ -4479,7 +4479,6 @@ void Tokenizer::createLinks2()
     if (isC())
         return;
 
-    const Token * templateToken = nullptr;
```

Comment:
No warning from Cppcheck about this variable? Do you know if there is a ticket?

Reply:
when I look in the source code I do not see this? A rebase is needed?
https://github.com/danmar/cppcheck/blob/main/lib/tokenize.cpp#L4498

Reply:
Yes, I didn't rebase this yet. Just commented on an open conversation.

I think this might not have been detected by cppcheck and requires a ticket. I think somebody else removed it.